types/messages.go

```diff
@@ -138,103 +130,206 @@ func (msg *SSVMessage) Decode(data []byte) error {
 
 // SSVMessage is the main message passed within the SSV network. It encapsulates the SSVMessage structure and a signature
 type SignedSSVMessage struct {
-	Signature  [256]byte // Created by the operator's network key
-	OperatorID OperatorID
-	Data       []byte
+	Signature  [][]byte     `ssz-max:"13,256"` // Created by the operators' key
+	OperatorID []OperatorID `ssz-max:"13"`
```

Comment:
since it's plural, i'd rename to Signatures and Signers or Operators

Reply:
I would rename now to make this PR makes sense
when we have a PR to drop decided we can rename again...

Unless we are sure @moshe-blox that we want to drop decided messages in this fork

Reply:
Just to remind everyone. In the last discussion the options were: Decided messages should either become post-consensus justifications,  or be emitted upon an RC messages, or be dropped entirely.

But this has nothing to do with this PR. We can rename in the meanwhile...

Reply:
> Unless we are sure @moshe-blox that we want to drop decided messages in this fork

it's yet to be decided (🤦‍♂️)

it's your call, but i think it's a quick rename and can be quickly undone if needed

Reply:
I agree with the renaming and it's easy to do. Updating it